pkg/volume/gce_pd/gce_util.go

```diff
@@ -58,6 +66,12 @@ func (util *GCEDiskUtil) AttachAndMountDisk(pd *gcePersistentDisk, globalPDPath
 	devicePath := ""
 	// Wait for the disk device to be created
 	for {
+		sdAfter, err := filepath.Glob("/dev/sd*")
+		if err != nil {
+			glog.Errorf("Error filepath.Glob(\"/dev/sd*\"): %v\r\n", err)
+		}
+		udevadmChangeToNewDrives(sdBefore, sdAfter)
+
```

Comment:
It should block only one pod and not the entire queue. I'm not clear on why this is a ```for {``` loop rather than a ```for numTries := 0; numTries < 10; numTries++ {``` loop.